android/src/main/java/com/instabug/reactlibrary/Constants.java

```diff
@@ -10,6 +10,6 @@ final class Constants {
     final static String IBG_ON_NEW_MESSAGE_HANDLER = "IBGonNewMessageHandler";
     final static String IBG_ON_NEW_REPLY_RECEIVED_CALLBACK = "IBGOnNewReplyReceivedCallback";
 
-    final static String IBGAPM_ON_NEW_W3C_FLAGS_UPDATE_RECEIVED_CALLBACK = "IBGAPMOnNewW3CFlagsUpdateReceivedCallback";
+    final static String IBGCore_ON_NEW_W3C_FLAGS_UPDATE_RECEIVED_CALLBACK = "IBGCoreOnNewW3CFlagsUpdateReceivedCallback";
```

Comment:
I think let's rename this to be prefixed with `IBG` instead of `IBGCore` to align with the rest of the constants, what do you think?

```suggestion
    final static String IBG_ON_NEW_W3C_FLAGS_UPDATE_RECEIVED_CALLBACK = "IBGOnNewW3CFlagsUpdateReceivedCallback";
```